### DISCUSSION TITLE
Color list is empty in the daily target 01/07/2023

### DISCUSSION THREAD

**Discussion by Vigowebs** [cssbattle/forum#347] [Bugs reports]
Hello team,

Daily target 01st July, 2023 color list is empty, I tried different browsers (Chrome, Firefox and Edge) but remains the same. Please see below screenshot for your reference.

![image](https://github.com/cssbattle/forum/assets/28861103/f4875d01-9ac8-46b1-b3a4-34de4c741471)

**Comment by chinchang** [comment]
You probably noticed already, but just for records here - it was fixed :)
